WcaOnRails/Gemfile

```diff
@@ -13,7 +13,7 @@ gem 'rails-i18n'
 gem 'activerecord-import'
 gem 'sassc-rails'
 gem 'uglifier'
-gem 'therubyracer'
+# gem 'therubyracer'
```

Comment:
Can you please just remove the gem instead of commenting it?

Reply:
Okay, I've removed it in the PR of another issue - https://github.com/thewca/worldcubeassociation.org/pull/5339. I'll try to debug this issue again tomorrow after making the changes you said.